### ISSUE TITLE
Markdown instead of TeX

### ISSUE THREAD

**Issue by degenerat3** [ritsec/constitution#8]
# We should switch to Markdown  
It really is the best move...  

### Why?  
Markdown is:
- intuitive to write/read
 - doesn't require any `makefile`
 - can generate pdfs 

### What's Wrong with TeX?  
|Feature   | TeX has it?   | Markdown has it? |
|---|---|---|
| gross syntax  | yes  | no |
| native on every™ platform | no | yes|
|  hard to render | yes   | no |
| easy to read  | no   | yes |
| requires a makefile | yes | no |  

### Bottom line  
Markdown is so widely used that even Github, that's right GITHUB (this platform) supports markdown in its issues. In fact, this entire issue has been composed using beautiful & fabulous markdown! Also, markdown supports 🔥 tons of emojis 🔥 which could potentially™ help increase democracy levels.

**Comment by s-newman** [comment]
better idea: use markdown, and then build a PDF using with pandoc like so:
`pandoc -o output.pdf input.md`
:thinking:
